ui/cap-react/src/antd/forms/widgets/SwitchWidget.js

```diff
@@ -25,7 +34,7 @@ const SwitchWidget = ({
   return (
     <Switch
       autoFocus={autofocus}
-      checked={typeof value === "undefined" ? false : value}
+      checked={schema.type === "string" ? value === "true" : value}
```

Comment:
Like this it works for `undefined` too (it defaults to `false` automatically). We still need to check for `value === "true"` when the type is `string` because otherwise we would be setting the switch as checked when the value is `"false"` (since it's a non-empty string).